### PR TITLE
Fix postman test, update logback to newer version

### PR DESCRIPTION
**Pull request by wing328**
### PR checklist
 
- [ ] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [ ] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [ ] Run the following to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
  ```
  ./mvnw clean package 
  ./bin/generate-samples.sh ./bin/configs/*.yaml
  ./bin/utils/export_docs_generators.sh
  ``` 
  (For Windows users, please run the script in [Git BASH](https://gitforwindows.org/))
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  IMPORTANT: Do **NOT** purge/delete any folders/files (e.g. tests) when regenerating the samples as manually written tests may be removed.
- [ ] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master` (upcoming 7.1.0 minor release - breaking changes with fallbacks), `8.0.x` (breaking changes without fallbacks)
- [ ] If your PR is targeting a particular programming language, @mention the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) members, so they are more likely to review the pull request.

### CODE REVIEW

File: modules/openapi-generator/src/test/java/org/openapitools/codegen/postman/PostmanCollectionCodegenTest.java [hunk 1]
```diff
@@ -786,7 +786,7 @@ public void testQueryParameterDescription() throws IOException {
         output.deleteOnExit();
 
         final CodegenConfigurator configurator = new CodegenConfigurator()
-                .setGeneratorName("postman-v2")
+                .setGeneratorName("postman-collection")
                 .setInputSpec("./src/test/resources/SampleProject.yaml")
```
Comment (wing328) [inline modules/openapi-generator/src/test/java/org/openapitools/codegen/postman/PostmanCollectionCodegenTest.java:790]: @gcatanese can you please take a look at this line when you've time? SampleProject.yaml is a config file but not an OpenAPI spec.

Comment (wing328) [reply modules/openapi-generator/src/test/java/org/openapitools/codegen/postman/PostmanCollectionCodegenTest.java:790]: Same issue in line 766

File: modules/openapi-generator/src/test/java/org/openapitools/codegen/postman/PostmanCollectionCodegenTest.java [hunk 2]
```diff
@@ -786,7 +786,7 @@ public void testQueryParameterDescription() throws IOException {
         output.deleteOnExit();
 
         final CodegenConfigurator configurator = new CodegenConfigurator()
-                .setGeneratorName("postman-v2")
+                .setGeneratorName("postman-collection")
```
Comment (gcatanese) [inline modules/openapi-generator/src/test/java/org/openapitools/codegen/postman/PostmanCollectionCodegenTest.java:789]: @wing328 Indeed it should be 
```
  .setGeneratorName("postman-collection")
  .setInputSpec("src/test/resources/3_0/postman-collection/SampleProject.yaml")
```

File: modules/openapi-generator/src/test/java/org/openapitools/codegen/postman/PostmanCollectionCodegenTest.java (hunk 1 above)
Comment (gcatanese) [reply modules/openapi-generator/src/test/java/org/openapitools/codegen/postman/PostmanCollectionCodegenTest.java:790]: Sorry, I made a mistake re-using a test on a local implementation (I did miss the CI failures though). The test file is `"src/test/resources/3_0/postman-collection/SampleProject.yaml"`

Comment (wing328) [reply modules/openapi-generator/src/test/java/org/openapitools/codegen/postman/PostmanCollectionCodegenTest.java:790]: can you please PM me via Slack to further discuss this?

**Comment by gcatanese** [comment]
See #17412 17412

**Pull request closed without merging**